rubocop-performance.gemspec

```diff
@@ -31,7 +31,7 @@ Gem::Specification.new do |s|
     'bug_tracker_uri' => 'https://github.com/rubocop-hq/rubocop-performance/issues'
   }
 
-  s.add_runtime_dependency('rubocop', '>= 0.90.0')
-  s.add_runtime_dependency('rubocop-ast', '>= 0.4.0')
-  s.add_development_dependency('simplecov')
+  s.add_runtime_dependency('rubocop', '~> 0.93.1')
```

Comment:
I think it's unnecessary to drop supporting 0.90 it because the gem not depending any new features since RuboCop 0.91 or higher and I also prefer `>= 0.90.0` to provide an opportunity to upgrade to 1.0.

Reply:
1. I think `1.0` can be incompatible with `0.9x.x` versions.
2. I guess `>= 0.90.0` includes `x.0` versions, which definitely will not compatible.

Reply:
As far as I know, no incompatibilities since RuboCop 0.90.0 have been found in the RuboCop Performance runtime.

Reply:
> As far as I know, no incompatibilities since RuboCop 0.90.0 have been found in the RuboCop Performance runtime.

Maybe, but what about the second point? Like RuboCop version 2.0, 3.0 and further.